### ISSUE TITLE
setSelect() after query(..) method

### ISSUE THREAD

**Issue by ivanghisleni** [FoolCode/SphinxQL-Query-Builder#156]
Hi, 

I have this problem, after generating sphinxql passing the strin to query() method, I want to reuse the same query changing the select fields but using getSelect and after setSelect will be ignored.

Is it the correct way?

Thaks

**Comment by oohnoitz** [comment]
I'm not exactly sure what you are asking. Can you rephrase your question or provide example code of what you are trying to attempt to do?
